server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -1056,7 +1077,7 @@ export function configureWebSocketServices(
 			 * [DocumentDeltaConnection.emitMessages](https://github.com/microsoft/FluidFramework/blob/ccb26baf65be1cbe3f708ec0fe6887759c25be6d/packages/drivers/driver-base/src/documentDeltaConnection.ts#L313C1-L321C4)),
 			 * but actual content is passed-thru and not decoded.
 			 */
-			(clientId: string, contentBatches: unknown[]) => {
+			(clientId: string, contentBatches: unknown[] | ISentSignalMessage[]) => {
```

Comment:
`unknown[] | ISentSignalMessage[]` should be the same as `unknown[]`, yes?

Reply:
Right, I was thinking of removing but I saw ODSP do this and I thought it could be good for readability/documentation purposes. Would it be good to explicitly call out the type here so future devs know that it is expected to use type guards to narrow down `unknown` to `ISentSignalMessage` before performing any operations?

Reply:
I think we are probably better served editing the comments than this type.
Client was a little different where we eventually want to change the type. Here, I don't think we'll ever want to change the type because it is off the wire and unknown.